tools/ci_build/github/azure-pipelines/linux-gpu-tensorrt-daily-perf-pipeline.yml

```diff
@@ -68,23 +68,39 @@ jobs:
           value: 21.07
         ${{ elseif eq(parameters.TrtVersion, '7.2.3.4') }}:
           value: 20.12
+      
```

Comment:
This is more of a clarification question for my own edification :)
Does the table below accurately summarize the relationship between the input parameters and the docker image used?

| RunNvidiaContainer   | BuildORT | Previous behavior              | New behavior |
|------------------------ | ---------- | ----------------------------- | --------------- |
| False                           | False    | `docker pull Dockerfile.ubuntu_*`<sup>1</sup> | `docker pull nvcr.io/nvidia/*` |
| False                           | True    | `docker build Dockerfile.ubuntu_*`<sup>1</sup> | `docker build Dockerfile.ubuntu_*`|
| True                           | False    | `docker pull nvcr.io/nvidia/*` | `docker pull nvcr.io/nvidia/*`|
| True                           | True    | `docker build Dockerfile.tensorrt` | `docker build Dockerfile.tensorrt`|

<sup>1</sup>: Not supported

Reply:
> 

Good point. RunNvidiaContainer/BuildORT false (1st row) should only run if trt version is up to date with master (i.e. 8.2 at the moment). 

The other rows are correct.